docs/source/workflows/predictors.rst

```diff
@@ -512,30 +516,40 @@ Missing ingredient properties can be handled one of three ways:
    Otherwise, the row will not be featurized.
    Use this option if you expect ingredient properties to be dense (always present) and would like to exclude rows when properties are missing.
-2. If ``impute_properties == True`` and no ``default_properties`` are specified, missing properties will be filled in using the average value across the entire dataset.
-   The average is computed from any row with data corresponding to the missing property, regardless of label or material type (i.e., the average is computed from all leaf ingredients and mixtures).
-3. If ``impute_properties == True`` and ``default_properties`` are specified, the specified property value will be used when an ingredient property is missing (instead of the average over the dataset).
+2. If ``impute_properties == True`` and no ``default_properties`` are specified,
+   missing properties will be filled in using the average value across the entire dataset.
+   For real-valued properties, this average is the mean over the training data,
+   while for categorical-valued properties, it is the distribution of property values in the dataset.
+   The average is computed from any row with data corresponding to the missing property,
+   regardless of label or material type (i.e., the average is computed from all leaf ingredients and mixtures).
+3. If ``impute_properties == True`` and ``default_properties`` are specified,
+   the specified property value will be used when an ingredient property is missing (instead of the average over the dataset).
    This allows complete control over what values are imputed.
    Default properties cannot be specified if ``impute_properties == False`` (because missing properties are not filled in).
 
-For example, say we add boric acid (a common antiseptic) as a possible ingredient to a saline solution but do not know its density.
+For example, say we add boric acid (a common antiseptic) as a possible ingredient to a saline solution,
+but do not know its density (a real-valued property) or its solubility in acetone (a categorical-valued property).
 Our leaf ingredient data might resemble:
 
-+---------------+----------------+
-| Ingredient id | Density (g/cc) |
-+===============+================+
-| water         | 1.0            |
-+---------------+----------------+
-| salt          | 2.16           |
-+---------------+----------------+
-| boric acid    | N/A            |
-+---------------+----------------+
++---------------+----------------+--------------------+
+| Ingredient id | Density (g/cc) | Acetone Solubility |
++===============+================+====================+
+| water         | 1.0            | Yes                |
++---------------+----------------+--------------------+
+| salt          | 2.16           | No                 |
++---------------+----------------+--------------------+
+| boric acid    | N/A            | N/A                |
++---------------+----------------+--------------------+
```

Comment:
Might use `boric acid` as `slightly soluble` versus water `soluble` and salt `insoluble`.